src/lib/yang/snabb-softwire-v2.yang

```diff
@@ -26,13 +21,117 @@ module snabb-softwire-v2 {
        This also removes the psid-map list and adds a new port-set
        container on the softwire container instead. This will help
        adding the softwires as well as bring it more inline with the
-       ietf-softwire schema.";
+       ietf-softwire schema.
+
+       The addition of /softwire-config/instance allows for configuring
+       multiple instances of the lwAFTR with a shared binding table and
+       other common configuration properties.";
+  }
+
+  revision 2016-11-04 {
+    description
+     "Initial revision.";
   }
 
   container softwire-config {
     description
      "Configuration for Snabb lwaftr.";
 
+    list instance {
+      description
+       "Provides configuration for specific instances of the lwAFTR.
+        These configuration options will only affect the specific lwaftr
+        with the given name specified in the name leaf. The other options
+        not present in this list are shared amongst all instances.";
+
+      key "name";
+
+      leaf name {
+        type string;
+        description
+         "Name of lwAFTR instance. This must be unique amongst the Snabb
+          processes on the system. This may be specified either here, in the
+          YANG configuration or via the command line when the lwAFTR is started.
+
+          The order of presidence for this leaf is as followers:
+          1. The name set on an already running lwAFTR instance via snabb set.
+          2. A command line option to specify the name upon starting the lwAFTR
+              instance (i.e. overriding this value).
+          3. The value here in the configuration when starting a lwaftr instance.
+
+          If no name is specified the lwaftr can be referred to using the PID of
+          the lwAFTR process on the system.";
+      }
+
+      container external-interface {
+        leaf ip {
+          type inet:ipv4-address;
+          mandatory true;
+          description
+           "L3 Address of the internet-facing network interface.  Used
+            when generating error messages and responding to ICMP echo
+            requests.";
+        }
+
+        leaf mac {
+          type yang:mac-address;
+          mandatory true;
```

Comment:
Ideally we will be able to pull this from the device itself in the future, FWIW.  The older 82599 driver didn't allow this but the newer one does, I think.